File: bizzfuzz/BizzFuzz.java
public class BizzFuzz {
    public static void main(String[] args) {
        for (int i = 1; i < 101; i++) {
            boolean mod4 = (i % 4 == 0);
            boolean mod7 = (i % 7 == 0);
            if (mod4)
                System.out.print("Bizz");
            if (mod7)
                System.out.print("Fuzz");
            if (!mod4 && !mod7)
                System.out.print(i);
            System.out.println();
        }
    }
}

File: bizzfuzz/bizzfuzz.py
import sys

for i in range(1, 101):
    mod4 = (i % 4 == 0)
    mod7 = (i % 7 == 0)
    if mod4:
        sys.stdout.write("Bizz")
    if mod7:
        sys.stdout.write("Fuzz")
    if not mod4 and not mod7:
        sys.stdout.write(str(i))
    print("")
